### ISSUE TITLE
HTML report: missing visible focus

### ISSUE THREAD

**Issue by jernst** [fediverse-devnet/feditest#414]
From Accessibility Report:

> The Degraded AssertionFailure (orange beams) interactive element doesn’t have a visible focus (are highlighted) when it receives focus using the keyboard.

References:
* https://www.w3.org/WAI/WCAG22/Understanding/keyboard
* https://www.w3.org/WAI/WCAG22/Understanding/focus-order
* https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

**Comment by jernst** [comment]
Moved to #435
